types/meteor-astronomy/meteor-astronomy-tests.ts

```diff
@@ -79,6 +79,7 @@ interface UserInterface extends Meteor.User {
     phone: string;
     phoneNumber: string;
     fullName: (param: string) => string;
+    profile: UserProfileInterface;
```

Comment:
Please remove this unrelated change.

Reply:
:( But the test were failing without this change!

Reply:
That's weird... OK.